pkg/server/container_create_windows.go

```diff
@@ -324,15 +324,27 @@ func (c *criService) generateContainerSpec(id string, sandboxID string, sandboxP
 				return nil, errors.Errorf(`pipe mount.HostPath '%s' not supported for LCOW`, formattedSource)
 			}
 		} else {
-			s, err := os.Stat(formattedSource)
+			s, err := c.os.Stat(formattedSource)
 			if err != nil {
 				return nil, errors.Wrapf(err, "failed to Stat mount.HostPath '%s'", formattedSource)
 			}
 			if !s.IsDir() {
 				ext := strings.ToLower(filepath.Ext(formattedSource))
 				if ext == ".vhd" || ext == ".vhdx" {
 					mo.Type = "virtual-disk"
-					mo.Options = append(mo.Options, "bind")
```

Comment:
Is this option already ignored, or why are we removing it here?

Reply:
The code was adding a duplicate option. All virtual-disk mounts get "bind" added below.

Reply:
I can send as a separate PR if you want.

Reply:
This is fine, was just curious.